src/Services/PopoverManager.vala

```diff
@@ -165,6 +168,12 @@ public class Wingpanel.Services.PopoverManager : Object {
         return current_indicator != null && current_indicator.base_indicator.code_name == entry.base_indicator.code_name;
     }
 
+    private void suppress_tooltip (Gtk.Widget display_widget, bool suppress = true) {
```

Comment:
See my comment about the function name -> https://github.com/elementary/wingpanel/pull/393#issuecomment-870670363

The reason I added this function to do the null check is because in https://github.com/elementary/wingpanel/blob/master/src/Widgets/IndicatorEntry.vala#L54 it appears the `display_widget` could be left null; so no harm in doing a null check here at worst.